src/fidesops/schemas/saas/saas_config.py

```diff
@@ -57,12 +58,19 @@ def check_reference_direction(
 
 
 class Strategy(BaseModel):
-    """General shape for swappable strategies (ex: auth, pagination, postprocessing, etc.)"""
+    """General shape for swappable strategies (ex: auth, pagination, etc.)"""
 
     strategy: str
     configuration: Dict[str, Any]
 
 
+class PostProcessorStrategyData(BaseModel):
+    """Data class for postprocessor strategy"""
+
+    strategy: str
+    configuration: StrategyConfiguration
```

Comment:
ok, so it looks like this `StrategyConfiguration`  is preventing the `configuration` from being persisted when we add the `saas_config` to the ConnectionConfig via `{{host}}/connection/{{saas_key}}/saas_config` - 

There are no actual fields on `StrategyConfiguration` so when validating the request, I assume it's just passed through as an empty dict by FastAPI/pydantic.

```
"postprocessors": [
                      {
                          "strategy": "unwrap",
                          "configuration": {}
                      },
                      {
                          "strategy": "filter",
                          "configuration": {}
                      }
                  ],
```

Reply:
I've just removed `PostProcessorStrategyData`